### DISCUSSION TITLE
Is there a minimize callback?

### DISCUSSION THREAD

**Discussion by alex3236** [fltk-rs/fltk-rs#1505] [Q&A] [answered]
I'm trying to build a app that can be hide to tray icon. Based on the sample, I've done most of it, but I want it hidden to tray icon when minimized (not just when the window closed). Any ideas?

**Comment by MoAlyousef** [comment, marked as answer]
Not really.
Maybe you could use something like:
```rust
window.resize_callback(|win, x, y, w, h| {
    if win.shown() && !win.visible() {
        // win is minimized
        // add to tray
    }
});
```

**Reply by alex3236** [reply]
thx
